src/main/kotlin/net/djvk/fireflyPlaidConnector2/sync/PolledSyncRunner.kt

```diff
@@ -81,12 +84,14 @@ class PolledSyncRunner(
                             executeTransactionSyncRequest(accessToken, cursorMap[accessToken], plaidBatchSize)
```

Comment:
Same comment as line 164. There are less null safe calls in this case, but being consistent is preferable.

Reply:
You could also just throw an exception here if you get null for initial cursor sync, but I'm fine either way with that.